api/src/main/java/edu/cornell/mannlib/vitro/webapp/controller/jena/JenaIngestController.java

```diff
@@ -659,7 +660,7 @@ private String processRenameResourceRequest(VitroRequest vreq) {
                   vreq.setAttribute("title","Rename Resource");
                   return RENAME_RESOURCE;
               } else {
-                  String result = doRename(oldNamespace, newNamespace);
+                  String result = doRename(oldNamespace, newNamespace, vreq);
```

Comment:
Do you have some vision that some properties from VitroRequest might be needed into doRename method? I mean why you are not passing only N3EditUtils.getEditorURI(vreq) as you did in the ChangeSet addRemoval method?

Reply:
fixed